apps/marketplace/actions/resetPasswordActions.js

```diff
@@ -26,6 +26,7 @@ export const sendResetPasswordEmail = values => (dispatch, getState) => {
   }).then(response => {
     if (response.error) {
       if (response.data && response.data.message && response.data.message.toLowerCase().includes('invalid csrf')) {
+        dispatch(setAuthState({ csrfToken: '' }))
```

Comment:
I think we should set to 
dispatch(setAuthState({ isAuthenticated: false, userType: '', csrfToken: '' }))

When it comes to this point , login already failed . We should clear up everything related to user